README.md

```diff
@@ -76,6 +77,18 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
 2. Expand the **Extensions** section and click **OpenAPI**.
 3. Clear the checkbox **Alphabetically sort contents of OpenAPI explorer outlines**.
 
+
+### Preview OpenAPI documentation
+
+OpenAPI files can be easily converted to a human readable HTML documentation describing how to use your API. Our extension supports
```

Comment:
"You can easily convert your OpenAPI files to human-readable HTML documentation that describes how to use your API."

Avoid passive voice, make user the focus, no article needed for "documentation", it's uncountable/mass noun.